applications/GeoMechanicsApplication/tests/cpp_tests/test_fluid_pressure.cpp

```diff
@@ -0,0 +1,43 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Gennady Markelov
+//
+
+#include "custom_retention/retention_law.h"
+#include "includes/checks.h"
+#include "testing/testing.h"
+
+using namespace Kratos;
+
+namespace Kratos::Testing
+{
+
+KRATOS_TEST_CASE_IN_SUITE(CalculateFluidPressureGivesCorrectResults, KratosGeoMechanicsFastSuite)
+{
+    Vector N(5);
+    N(0) = 1.0;
+    N(1) = 2.0;
+    N(2) = 3.0;
+    N(3) = 4.0;
+    N(4) = 5.0;
```

Comment:
I believe these assignments can be simplified by using operator `<<=` (after including header `boost/numeric/ublas/assignment.hpp`):
```suggestion
    N <<= 1.0, 2.0, 3.0, 4.0, 5.0;
```

Reply:
Many thanks! Copilot was not able to point to the header file.